person.rb

```diff
@@ -1,5 +1,6 @@
-class Person
+class Person < Nameable
```

Comment:
- [ ] Please make sure to use require_relative from the Person class as well otherwise your application will not work.